pages/partials/examQuestionAvailablePoints.ejs

```diff
@@ -19,7 +19,7 @@
     <% } %>
 
     <!-- Create popover with info about what will happen on next submission -->
-    <button type="button" class="btn btn-xs" data-toggle="popover" data-html="true" data-content="<%= include('examQuestionAvailablePointsInfo', {bestScore: best, pointsAvailable: points_list[0], triesRemaining: points_list.length}) %>" data-placement="auto" data-trigger="focus" data-container="body" tabindex="0">
+    <button type="button" class="btn btn-xs" data-toggle="popover" data-html="true" data-content="<%= include('examQuestionAvailablePointsInfo', {bestScore: best, pointsAvailable: points_list[0], currentWeight: currentWeight, triesRemaining: points_list.length}) %>" data-placement="auto" data-trigger="focus" data-container="body" tabindex="0">
```

Comment:
I wasn't sure whether this change was strictly necessary?

Reply:
I believe that it's not necessary, but it doesn't really hurt either.